src/include/cell.h

```diff
@@ -144,10 +144,10 @@ struct __wt_cell {
     uint8_t __chunk[1 + 1 + 1 + 7 * WT_INTPACK64_MAXSIZE + WT_INTPACK32_MAXSIZE];
 };
 
-/* AUTOMATIC FLAG VALUE GENERATION START */
+/* AUTOMATIC FLAG VALUE GENERATION START 0 */
 #define WT_CELL_UNPACK_OVERFLOW 0x1u            /* cell is an overflow */
 #define WT_CELL_UNPACK_TIME_WINDOW_CLEARED 0x2u /* time window cleared because of restart */
-                                                /* AUTOMATIC FLAG VALUE GENERATION STOP */
+/*AUTOMATIC FLAG VALUE GENERATION STOP 8 */
```

Comment:
Did this get generated properly? A space got removed.

Reply:
thanks! I accidentally removed the space, when verifying the changes.